compiler/src/iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -1086,12 +1086,12 @@ LogicalResult initSPIRVLaunchConfig(ModuleOp module) {
       rootOperation = computeOp;
     }
 
-    // Propogate the `lowering_config` attribute to the other ops.
-    // TODO(ravishankarm, antiagainst): This is a very specific use (and
-    // fragile). In general, this should not be needed. Things are already tiled
-    // and distributed. The rest of the compilation must be structured to either
-    // use `TileAndFuse` or they are independent configurations that are
-    // determined based on the op.
+    // Propogate the `lowering_config` attribute to the other ops. This is
+    // necessary because we typically use the last op as the anchor for tiling
```

Comment:
FYI, such a logic already exists in `TileAndDistributeToWorkgroups`. Could that be usable here. It would be ideal to have a single `LoweringConfig` operation and only use that instead of propagation.

Reply:
Thanks! I exposed the utility function there.